package.json

```diff
@@ -45,6 +42,7 @@
     "whatwg-fetch": "2.0.3"
   },
   "devDependencies": {
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
```

Comment:
This is probably required to compile the project, therefore it should be under `dependencies`.

Reply:
I fixed this already.